tests/app/test_utils.py

```diff
@@ -0,0 +1,55 @@
+from app.utils import (display_list,
```

Comment:
Again, I don't think we've ever agreed on this, but things that are tested usually are imported after the test tools.